tests/conftest.py

```diff
@@ -40,6 +40,7 @@ def pytest_addoption(parser):
     parser.addoption(
         "--lab",
         action="store",
+        default="ci",
```

Comment:
This means that anyone running the tests locally without being aware of the `--lab` option (which is usually the case until you see the error the first time) will actually use the lab of the ci, potentially creating conflicts, while not being aware of it.  How about rather support setting the option via environment as well and set this one in the ci? (I think the ci templates we have can support this?)

Reply:
I see that you actually went for the suggestion of @bartv, I don't know how close to it the solution should be.  The current implementation might as well be a temporary patch to fix the ci, and we can implement the env variable support later in a new ticket.